docs/quick-start.MD

```diff
@@ -39,6 +39,69 @@ kubectl get pods
 
 ```
 
+
+## Installing by Kustomize
+
+You can install operator using [Kustomize](https://kubernetes-sigs.github.io/kustomize/) by pointing to the remote kustomization file.
+
+```yaml
```

Comment:
what do you think, is it better to use
cat << EOF >> file_name
...content
EOF
 
 Or just place desired filename to comment?